src/widgets/fancytabwidget.h

```diff
@@ -43,8 +44,8 @@ class FancyTabWidget : public QTabWidget {
         void setBackgroundPixmap(const QPixmap& pixmap);
         void addSpacer();
 
-        void loadSettings(const char *);
-        void saveSettings(const char *);
+        void loadSettings(QSettings&);
```

Comment:
Do not use non-const references, you probably just want to pass-by-value here as with most `QObjects`

Reply:
Just to be clear, you're talking about only this change (to `loadSettings`, not about, e.g., `saveSettings` right beneath it)?

Reply:
Both

Reply:
Sorry, then I'm confused. `saveSettings` changes the `QSettings` object (in set). It can't be passed const. What am I missing?

Reply:
Ah, I see what you're doing here now. It should be:
```c++
void loadSettings(const QSettings&);
void saveSettings(QSettings*);
```

Reply:
Gotcha! I'll probably be able to get these resolved tonight. Thanks for your comments!